x-pack/legacy/plugins/siem/public/pages/hosts/index.tsx

```diff
@@ -44,205 +36,51 @@ export const HostsContainer = React.memo<Props>(({ url }) => (
-                    from={from}
-                    to={to}
-                    setQuery={setQuery}
-                    isInitializing={isInitializing}
-                    children={HostsQueryTabBody}
-                    detailName={props.match.params.detailName}
-                  />
-                )}
-              />
-              <Route
-                path={`${hostsPagePath}/:detailName/:tabName(${HostsTableType.authentications})`}
-                render={() => (
-                  <HostDetailsBody
-                    deleteQuery={deleteQuery}
-                    from={from}
-                    to={to}
-                    setQuery={setQuery}
-                    isInitializing={isInitializing}
-                    detailName={props.match.params.detailName}
-                    children={AuthenticationsQueryTabBody}
-                  />
-                )}
-              />
-              <Route
-                path={`${hostsPagePath}/:detailName/:tabName(${HostsTableType.uncommonProcesses})`}
-                render={() => (
-                  <HostDetailsBody
-                    deleteQuery={deleteQuery}
-                    from={from}
-                    to={to}
-                    setQuery={setQuery}
-                    isInitializing={isInitializing}
-                    detailName={props.match.params.detailName}
-                    children={UncommonProcessQueryTabBody}
-                  />
-                )}
-              />
-              <Route
-                path={`${hostsPagePath}/:detailName/:tabName(${HostsTableType.anomalies})`}
-                render={() => (
-                  <HostDetailsBody
-                    deleteQuery={deleteQuery}
-                    from={from}
-                    to={to}
-                    setQuery={setQuery}
-                    isInitializing={isInitializing}
-                    detailName={props.match.params.detailName}
-                    children={AnomaliesQueryTabBody}
-                  />
-                )}
-              />
-              <Route
-                path={`${hostsPagePath}/:detailName/:tabName(${HostsTableType.events})`}
-                render={() => (
-                  <HostDetailsBody
-                    from={from}
-                    to={to}
-                    setQuery={setQuery}
-                    isInitializing={isInitializing}
-                    detailName={props.match.params.detailName}
-                    children={EventsQueryTabBody}
-                  />
-                )}
-              />
-            </>
+            <HostDetails
+              hostDetailsPagePath={hostDetailsPagePath}
+              detailName={props.match.params.detailName}
+              from={from}
+              to={to}
+              setQuery={setQuery}
+              isInitializing={isInitializing}
+              deleteQuery={deleteQuery}
+            />
           )}
         />
         <Route
           path={`${url}/:detailName`}
```

Comment:
When referring to the path of hosts details page, we've got two expressions here in this file, 
like `${url}/:detailName` and `hostDetailsPagePath`, what about let's align with `hostDetailsPagePath`?